srv/_grains/platform.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
+
+import os
+
+def add_lr_serial_number():
+    """
+    Populates lr-serial-number to grains
+    """
+
+    grains = {}
+    grains["lr-serial-number"] = None
+
+    filename = "/opt/seagate/lr-serial-number"
```

Comment:
Since there is an issue with reading pillar values in grains, let's maintain the file name hard-coding here.
In that case, define the filename as a CONST variable globally.

Reply:
Done.